tardis/model/parse_input.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd
 from astropy import units as u
 
+from tardis import constants as const
```

Comment:
Duplicate import

Reply:
there is just one important. Not sure what you mean.

Reply:
https://github.com/tardis-sn/tardis/blob/0c36b9bec4ba1c873d448f694ffc395527ed8bfd/tardis/model/parse_input.py#L9

This line needs to be removed

Reply:
sorry it blurred with the next one. definitely.